File: JavaToCSharpConverter/Source/RescueZStack.java
package org.Rescue.rjni;
import java.lang.*;
/*****************************************************************

    RescueZStack.java

    Rod Hanks   Sep  5 2005

******************************************************************/
public class RescueZStack extends RescueGeometryObject
{


  protected RescueZStack(long ndxIn)
  {
    nativeNdx = ndxIn;
  }

  public void dispose()
  {
    Delete_RescueZStack(nativeNdx);
  }

  private native void Delete_RescueZStack(long ndx);
}

File: JavaToCSharpConverter/Output/RescueZStack.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace RescueJ
{
public class RescueZStack : RescueGeometryObject
{


  protected RescueZStack(long ndxIn)
  {
    nativeNdx = ndxIn;
  }

  public void dispose()
  {
    Delete_RescueZStack(nativeNdx);
  }

}

}